util/grpc/src/grpcio_extensions.rs

```diff
@@ -77,7 +77,7 @@ pub trait ConnectionUriGrpcioServer {
 }
 
 impl ConnectionUriGrpcioServer for ServerBuilder {
-    fn bind_using_uri(self, uri: &impl ConnectionUri, logger: Logger) -> Self {
+    fn build_using_uri(self, uri: &impl ConnectionUri, logger: Logger) -> Result<Server> {
         if uri.use_tls() {
```

Comment:
Unfortunately `Uri::use_tls()` returns a boolean, based on the scheme (i.e. `if scheme_string == Self::SECURE_SCHEME { true } else { false }`), so what you'd really want is a new method on the URI that could return a tuple, so it would look more like:

```rust
if let (tls_chain_path, tls_key_path) = uri.tls_config() {
    /* ... */
}
```